python/kserve/kserve/metrics.py

```diff
@@ -0,0 +1,12 @@
+from prometheus_client import Histogram
+import os
+
+PROM_LABELS = ['model_name']
+PRE_HIST_TIME = Histogram('request_preprocess_seconds', 'pre-process request latency', PROM_LABELS)
+POST_HIST_TIME = Histogram('request_postprocess_seconds', 'post-process request latency', PROM_LABELS)
+PREDICT_HIST_TIME = Histogram('request_predict_seconds', 'predict request latency', PROM_LABELS)
+EXPLAIN_HIST_TIME = Histogram('request_explain_seconds', 'explain request latency', PROM_LABELS)
+
+
+def get_labels(model_name):
```

Comment:
we can add more in future if we want, for now this is just cleaner to have so we can insert like **prom_labels in models.py